mono/utils/mono-context.c

```diff
@@ -326,6 +342,22 @@ mono_monoctx_to_sigctx (MonoContext *mctx, void *sigctx)
 	context->R13 = mctx->gregs [AMD64_R13];
 	context->R14 = mctx->gregs [AMD64_R14];
 	context->R15 = mctx->gregs [AMD64_R15];
+	memcpy (&(context->Xmm0), &(mctx->fregs [AMD64_XMM0]), sizeof (MonoContextSimdReg));
```

Comment:
Ditto. Just on mempcy.
Also memcpy the gregs all at once maybe.

Reply:
Also excess parens.
Can be `memcpy (&context->Xmm0, &mctx->fregs [AMD64_XMM0], sizeof (Mono..));`
also, ideally, if the types match:
`context->Xmm0 = mctx->fregs [AMD64_XMM]);`

I realize that might not work. `memcpy` might be glossing over slightly different types.

Reply:
Same as above and will keep assignment this way to be consistent with how other platforms do it for MonoContext .

Reply:
It's different types (but underluing representation should be the same) so direct assignment won't work, that's why I did memcpy just doing a straight transfer of underlying memory and since memcpy is an intrinsic it will be lowered to the same underlying assembler code:

movups      xmm0,xmmword ptr [rcx+1A0h]  
movups      xmmword ptr [rdx+90h],xmm0